apps/editor/src/app/coral-island-save-game/coral-island-save-game.type.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Save game type with extracted types for the editor. Type does *not* include all types, only the ones essential for
+ * the purpose of this editor.
+ */
+export type CoralIslandSaveGame = {
+  root: {
+    properties: {
+      SaveData_0: {
+        Struct: {
+          value: {
+            Struct: {
+              currentDate_0: {
+                Struct: {
+                  value: {
+                    Struct: {
+                      day_0: {
+                        Int: number;
+                      };
+                    };
+                  };
+                };
+              };
+              players_0: {
+                Array: {
+                  value: {
+                    Struct: {
+                      value: {
+                        Struct: {
+                          playerInfo_0: {
+                            Struct: {
+                              value: {
+                                Struct: {
+                                  Name_0: {
+                                    Str: 'Alex';
```

Comment:
@koenigderluegner This should be `string`.
I also I guess we could make utility types to replace all those nested Struct.value, Array.value etc. 
Now, I don't know if it would make the path easier to read, lol.